lua/core/default_config.lua

```diff
@@ -76,20 +76,9 @@ M.plugins = {
       packer = {
          init_file = "plugins.packerInit",
       },
-      autopairs = { loadAfter = "nvim-cmp" },
```

Comment:
why this ?

Reply:
I just realized that autopairs is what is breaking some stuff.  I wanted to remove the "loadafter" stuff since it can be overriden in the new format but there must be something which still references it in the init.lua. I'll fix it real quick.